docs/api/f_api.rst

```diff
@@ -238,6 +238,8 @@ __________________________
 Reinforcement Learning
 **********************
 
+Similar to other reinforcement learning frameworks such as [spinning up](https://spinningup.openai.com/en/latest/) from OpenAI or [stable baselines](https://stable-baselines3.readthedocs.io/en/master/), we distinguish between on-policy and off-policy algorithms since those two types require different APIs.
```

Comment:
The new on-policy APIs need to be added to the Fortran documentation (and also the C API section).

Reply:
I have added those, please check.